pkg/pillar/cmd/zedrouter/ipsec.go

```diff
@@ -746,8 +740,7 @@ func ipSecConfigCreate(vpnConfig types.VpnConfig) error {
 	if err := ipSecConfigFileWrite(filename, writeStr); err != nil {
 		return err
 	}
-	cmd := exec.Command("chmod", "600", filename)
-	_, err := cmd.Output()
+	_, err := base.ExecCommand(log, false, false, "chmod", "600", filename)
```

Comment:
Better to use os package instead of Exec

Reply:
Done.